OpusCapita/service-base-ui
# (BNAPP-1714)handle username if so long

# Description

Fixes [BNAPP-1714](https://opuscapita.atlassian.net/browse/BNAPP-1714).

<ins>Please include a summary of the change (current and new behavior):</ins>


Shortcut for too long username.

<br/>

### 🛠️ Type of change

Please check the type of change your PR introduces:

<br/>


### Does this PR introduce a breaking change?

<br/>

# How to test the change locally
Just log in as user with long username E.g. `manager.commodity_de@acmemax.com`, then open user window and check results


*✍️ replace this line ✍️*






<br/>

# Other informations
